docs/source/guidelines.rst

```diff
@@ -351,6 +351,14 @@ If the recipe was created with ``conda skeleton cpan``, the tests are
 likely sufficient. Otherwise, test the import of modules (see the
 ``imports`` section of the ``meta.yaml`` files in above examples).
 
+Additionally, if the recipe was created with ``conda skeleton cpan``, several modifications
+are necessary to satisfy bioconda policies:
+
+- remove the ``bld.bat`` script
+- remove the ``source/fn`` entry in ``meta.yaml``
+- the ``requirements/build`` keyword in ``meta.yaml`` should be changed to
```

Comment:
Hu, is this still the case with the latest conda version?

Reply:
ping @jvolkening

Reply:
As of 4.5.11, yes to all three. It also creates a version subdir.

Reply:
Also generates `version: {{version}}` which is broken (`0.80` will be parsed as `0.8` by yaml).